components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -110,9 +110,11 @@ $script:pkg_expose = @()
 # An associative array representing configuration data which should be gossiped to peers.
 $script:pkg_exports = @{}
 # The user to run the service as
-$script:pkg_svc_user = "hab"
+$script:pkg_svc_user = ""
 # The group to run the service as
-$script:pkg_svc_group = "$pkg_svc_user"
+# This is not actually used on Windows but
+# it needs to exist
```

Comment:
because the `Pkg` struct expects both user and group to be populated otherwise it handles neither. Ideally we would have a windows only implementation that just ignored the group. That would be trivial to add but wisks backward compatibility with new packages without `SVC_GROUP` and older supervisors.

Reply:
I added ^^^ to the comments